tachyon/zk/plonk/circuit/region_shape.h

```diff
@@ -69,7 +69,7 @@ class RegionShape : public Region<F>::Layouter {
     row_count_ = std::max(row_count_, offset + 1);
   }
 
-  const size_t region_index_ = 0;
+  size_t region_index_ = 0;
   absl::flat_hash_set<RegionColumn> columns_;
   size_t row_count_ = 0;
```

Comment:
Why do you remove const here? Isn't it a constant?

Reply:
In code like `SlotInBiggestAdviceFirst` function where it calls `std::sort` on sorted_regions, it requires copy assignment operator, but since `region_index_` was declared as const, the copy assignment operator of `RegionShape` was implicitly deleted. Should I explicitly define copy assignment operator over `RegionShape`?